addon/components/osf-paginator/component.js

```diff
@@ -24,6 +27,13 @@ import layout from './template';
 export default Ember.Component.extend({
     layout,
     currentPage: 1,
+    init() {
+        this._super(...arguments);
+        Ember.deprecate('osf-paginator is now deprecated. Use pagination-pager instead', false, {
+            id: 'osf-paginator',
+            until: '0.1.0'
```

Comment:
Thanks for adding this!  This is a good habit to get into as we start removing old stuff.

I suggest changing this to `1.0.0` as follows.

From the deprecation docs, this parameter should be a future release, rather than specifying the current version number:
> until String
The version of Ember when this deprecation warning will be removed.

We will be removing an entire component, which is not a small change! According to the [semver](http://semver.org/) specification that this addon follows, we should only make backwards-incompatible changes on a major version, eg `0.1.0 -> 1.0`.

As written, this warning implies that the component has already been removed ("will be gone by version 0.1"), which could be confusing.